app/services/iiif_print/manifest_builder_service_behavior.rb

```diff
@@ -144,7 +144,9 @@ def get_solr_hits(ids)
         query = "id:(#{paged_ids.join(' OR ')})"
         results += IiifPrint.solr_query(
           query,
-          { fq: "-has_model_ssim:FileSet", rows: paged_ids.size, method: :post }
+          fq: "-has_model_ssim:FileSet",
```

Comment:
This works, but will likely cause problems for certain configurations.  However, those certain configurations are not yet present.